tools/gendocs/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	gptscript "github.com/gptscript-ai/gptscript/pkg/cli"
+	"github.com/spf13/cobra/doc"
+)
+
+const fmTemplate = `---
+title: "%s"
+---
+`
+
+func main() {
+	cmd := gptscript.New()
+	cmd.DisableAutoGenTag = true
+
+	files, err := filepath.Glob("docs/docs/100-reference/01-command-line/gptscript_*.md")
+	if err != nil {
+		log.Fatal(err)
```

Comment:
looks like the import for `log` is missing.

Reply:
hmm? it's up on line 5

Reply:
Ah, yup. Should stop reviewing from my phone!